deployment/build-open-source-dist.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# This script packages your project into an open-source solution distributable 
+# that can be published to sites like GitHub.
+#
+# Important notes and prereq's:
+#   1. The initialize-repo.sh script must have been run in order for this script to
```

Comment:
does `initialize-repo.sh` exist? I don't see it.

Reply:
This part's not needed. Removing.